install.sh

```diff
@@ -347,6 +352,32 @@ get_install_dir() {
     printf %s "$_dir"
 }
 
+# Retrieve the version from the command-line arguments
+# Accepts flag: --version 1.1.0, --version v1.1.0, --version latest
+# Default: latest
+get_version() {
+    local _version
+    _version="latest"
+
+    while [[ $# -gt 0 ]]; do
+        case "$1" in
+        --version)
+            _version="$2"
+            shift 2
+            ;;
+        *)
+            shift
+            ;;
+        esac
+    done
+
+    if [[ $_version == v* ]] || [[ $_version == "latest" ]]; then
+        printf %s "$_version"
+    else
+        printf 'v%s' "$_version"
+    fi
```

Comment:
this reads better to me if you do it like
```suggestion
    if [[ $_version == [0-9]* ]]; then
        _version="v$_version"
    fi
    printf %s "$_version"
```

Reply:
used your logic + few changes to address shellcheck